extensions/arc/README.md

```diff
@@ -1,3 +1,5 @@
 # Arc
 
-Search and quickly navigate Arc's history and open tabs. You can open recent tabs in Arc or Little Arc as well as copy the websites to quickly share them.
+Search and quickly navigate Arc's history and open tabs. You can open recent tabs in Arc or Little Arc as well as copy websites to quickly share them.
+
+Experience a calmer, more personal internet in this browser designed for you. Let go of the clicks, the clutter, the distractions with the Arc browser. If you don't have Arc yet, download it at [arc.net](https://arc.net).
```

Comment:
Not sure how Raycast folks think about the extensions marketplace, but personally I don't feel this is the right place to market the external piece of software. I suspect that discovering new software to use via extensions is not a common use case. I also just feel that the strong marketing message feels out of place in this context.

Reply:
Changed this to just link out from the existing first paragraph